### PR TITLE
DOCSP-42372: Change username and password placeholders

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42372>
Staging - <https://preview-mongodbmcmorisi.gatsbyjs.io/java/DOCSP-42372-username-password/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?

### CODE REVIEW

File: source/fundamentals/indexes.txt [hunk 1]
```diff
@@ -426,31 +424,28 @@ in the {+mdb-server+} manual.
...
 Unique indexes ensure that the indexed fields do not store duplicate values. By default, MongoDB creates a unique index
 on the ``_id`` field during the creation of a collection. To create a unique index, specify the field or combination of
 fields that you want to prevent duplication on and set the ``unique`` option to ``true``.
 
 The following example creates a unique, descending index on the ``theaterId`` field:
 
-.. literalinclude:: /includes/fundamentals/code-snippets/IndexPage.java
-   :language: java
+.. literalinclude:: 
+   :language: csharp
```
Comment (norareidy) [inline source/fundamentals/indexes.txt:434]: I: This code block is no longer rendering on the staging site - I think you can just change it back to how it was

File: source/usage-examples.txt [hunk 2]
```diff
@@ -66,7 +66,7 @@ to learn how to allow connections to your instance of Atlas and to find the
 :manual:`connection string </reference/connection-string/>` you use to replace the
 ``uri`` variable in usage examples. If your instance uses
 :manual:`SCRAM authentication </core/security-scram/>`, you can replace
-``<user>`` with your username, ``<password>`` with your password, and
+``<db_username>`` with your username, ``<db_password>`` with your password, and
```
Comment (norareidy) [inline source/usage-examples.txt:69]: S:
```suggestion
``<db_username>`` with your database username, ``<db_password>`` with your database password, and
```

**Review by norareidy — APPROVED** [review]
lgtm + a couple things!

File: source/fundamentals/indexes.txt (hunk 1 above)
Comment (mcmorisi) [reply source/fundamentals/indexes.txt:434]: Not sure why so many unrelated changes/modifications/deletions got included in this commit, let me investigate to make sure I pulled in the changes correctly

**Pull request merged**